### ISSUE TITLE
Incorporate sentiment into model

### ISSUE THREAD

**Issue by mpaluta** [mpaluta/headline_generation#10]
- [x] Incorporate sentiment data or package into model
- [ ] Try pre-processing approach
- [ ] Try custom loss function in tensorflow
Ex. 
```
def augmented_ROUGE(actual, predicted):
    loss = ROUGE(actual, predicted) + sentiment_score(actual, predicted)
    return loss

# ...
model.compile(loss=augmented_ROUGE)
```
reference: https://github.com/tyu0912/MIDS_W207_facial_recognition/blob/master/FinalProject%20-%20Facial%20Recognition.ipynb
